src/components/Popups/ReplyField/ItemRow.scss

```diff
@@ -0,0 +1,7 @@
+@import '~box-ui-elements/es/styles/variables';
+
+.ba-ItemRow-email {
```

Comment:
It looks like font color, letter spacing, line height, and spacing are all a bit off from the designs for both name and email. Can we make sure these are as pixel perfect as possible?

Reply:
It looks like the user's name is also bold in the rendered link after a selection is made.